jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -606,7 +606,8 @@ properties:
               # NIRSpec
               NRS_AUTOFLAT, NRS_AUTOWAVE, NRS_BRIGHTOBJ, NRS_CONFIRM, NRS_DARK,
               NRS_FIXEDSLIT, NRS_FOCUS, NRS_IFU, NRS_IMAGE, NRS_LAMP, NRS_MIMF,
-              NRS_MSASPEC, NRS_MSATA, NRS_TACONFIRM, NRS_TACQ, NRS_TASLIT, NRS_WATA,
+              NRS_MSASPEC, NRS_MSATA, NRS_TACONFIRM, NRS_TACQ, NRS_TASLIT,
+              NRS_VERIFY, NRS_WATA,
```

Comment:
Thanks for maintaining the alphabetical listing!

Reply:
But of course! Is there any other way?